### DISCUSSION TITLE
Corrupted excel file

### DISCUSSION THREAD

**Discussion by janpernik** [jxlsteam/jxls#328] [Q&A] [answered]
Hi
we are using version 2.14 and sometimes it happens that JXLS does not recoginze JX: commands in notest and does not generate and comments stays untouched. Something like corrupted file, but with no error from excel or JXLS/POI.
Example in attachment.
[template.xlsx](https://github.com/user-attachments/files/16390698/template.xlsx)
Any hint what to do?

**Comment by SoltauFintel** [comment, marked as answer]
Use only Microsoft Excel for creating the template files. Recreate the template from scratch.

Could it be due to the data? e.g. an empty list, strange values or strange formulas

Do you use streaming?

**Reply by janpernik** [reply]
Template file is created only MS Excel 2019. No, it is not streaming. Recreating from scratch is working but too many work. Would be nice to find where the file is corrupted.
